docs/userguides/accounts.md

```diff
@@ -57,6 +57,21 @@ If your testing provider supports this feature, it is possible to directly set t
 account.balance += int(1e18)  # Gives `account` 1 Ether
 ```
 
+### Default Sender Support
+
+In order to eliminate the usage of sender in contract calls, you can use `use_sender` context manager.
+
+```python
+with test_accounts.use_sender(0):
+  contract.myFunction(1)
+
+with test_accounts.use_sender("address"):
+  contract.myFunction(1)
+
+with test_accounts.use_sender(TestAccount):
+  contract.myFunction(1)
```

Comment:
This might be clearer:
```suggestion
def test_example(accounts):
    with accounts.use_sender(a):  # a is a `TestAccount` class
        contract.myFunction(1)
```